implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -160,16 +164,41 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
     @ConfigProperty(name = "mp.jwt.verify.requireiss", defaultValue = "true")
     private Optional<Boolean> mpJwtRequireIss;
 
+    /**
+     * @since 1.2
+     */
+    @Inject
+    @ConfigProperty(name = Names.TOKEN_HEADER)
+    private Optional<String> mpJwtTokenHeader;
```

Comment:
Both this and then next property should be just Strings with the default properties